pyanaconda/core/constants.py

```diff
@@ -493,6 +493,14 @@ class DisplayModes(Enum):
     GROUP_PACKAGE_TYPE_CONDITIONAL,
 ]
 
+PACKAGE_KERNELS = [
+    "kernel",
+    "kernel-rt",
+    "kernel-redhat",
+    "kernel-64k",
+    "kernel-redhat-64k",
+]
```

Comment:
Can we please _not_ hardcode this into the Anaconda code? Aside from the fact that Fedora and RHEL aren't the only distributions that use Anaconda (and thus hardcoding kernel package names becomes really problematic), the names of package variants will differ in Fedora and RHEL/CentOS.

As an example, Fedora Asahi 16k kernel vs standard Fedora 4k kernel for Apple Silicon. Another example is CentOS Hyperscale's kernel vs the standard RHEL kernel.

We don't know what we're going to do in the ecosystem now and the future, but we _do_ know that hardcoding things is a seriously bad idea.

cc: @davide125

Reply:
@poncovka

Reply:
@jkonecny12 will verify that, but it looks like we won't need this list after all. See https://github.com/rhinstaller/anaconda/pull/4689#discussion_r1188889907.